src/telemetry/graph.jsx

```diff
@@ -32,6 +45,7 @@ export default class TelemetryContainer extends React.Component {
       this.graphSubtitleEl.textContent = graphData.description;
       this.graphEvolutionsTimeline(graphData, this.graphEl);
     } catch (error) {
+      this.setState({ errorFlag: true });
```

Comment:
I would rename `errorFlag` to something that more clearly denotes it's a boolean, such as `showErrorMessage`.

Reply:
I suggest renaming `errorFlag` to plain `error` so any value can be assigned, specifically a more detailed description of what happened.  `error != null` is an error.

Reply:
@klahnakoski got it. I will do it in the next push. PR is ready for review with requested changes.

cc: @sarah-clements

Reply:
> I suggest renaming `errorFlag` to plain `error` so any value can be assigned, specifically a more detailed description of what happened. `error != null` is an error.

True, I didn't suggest this previously since it seemed only 5xx errors would be relevant here and then it's a matter of messaging. So a generic "Something went wrong, please try again later." vs "NetworkError when attempting to fetch resource".